go/pkg/tree/tree.go

```diff
@@ -27,8 +27,12 @@ type treeNode struct {
 }
 
 type fileNode struct {
-	Chunker              *chunker.Chunker
-	IsExecutable         bool
+	Chunker      *chunker.Chunker
+	IsExecutable bool
+}
+
+type fileSysNode struct {
+	file                 *fileNode
```

Comment:
`File`?

Reply:
Done (but i feel like it should be the other way around, since these are all private types)

Reply:
Yeah, I agree with you.

Reply:
OK, I took it upon myself to change all these to lower case. @ola-rozenfeld  @gkousik